### PR TITLE
Add DuoRC Dataset

### PR DESCRIPTION
**Pull request by gchhablani**
Hi,

DuoRC SelfRC is one type of the [DuoRC Dataset](https://duorc.github.io/). DuoRC SelfRC is a crowdsourced Abstractive/Extractive Question-Answering dataset based on Wikipedia movie plots. It contains examples that may have answers in the movie plot, synthesized answers which are not present in the movie plot, or no answers.  I have also added ParaphraseRC - the other type of DuoRC dataset where questions are based on Wikipedia movie plots and answers are based on corresponding IMDb movie plots.

Paper :  [https://arxiv.org/abs/1804.07927](https://arxiv.org/abs/1804.07927)

I want to add this to 🤗 datasets to make it more accessible to the community. I have added  all the details that I could find. Please let me know if anything else is needed from my end.

Thanks,
Gunjan

### CODE REVIEW

File: datasets/duorc/README.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+[MIT License](https://github.com/duorc/duorc/blob/master/LICENSE)
+
+### Citation Information
+
+@inproceedings{DuoRC,
+author = { Amrita Saha and Rahul Aralikatte and Mitesh M. Khapra and Karthik Sankaranarayanan},
+title = {{DuoRC: Towards Complex Language Understanding with Paraphrased Reading Comprehension}},
+booktitle = {Meeting of the Association for Computational Linguistics (ACL)},
+year = {2018}
+}
```
Comment (lhoestq) [inline datasets/duorc/README.md:161]:
````suggestion
```
@inproceedings{DuoRC,
author = { Amrita Saha and Rahul Aralikatte and Mitesh M. Khapra and Karthik Sankaranarayanan},
title = {{DuoRC: Towards Complex Language Understanding with Paraphrased Reading Comprehension}},
booktitle = {Meeting of the Association for Computational Linguistics (ACL)},
year = {2018}
}
```
````

File: datasets/duorc/README.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** https://duorc.github.io/
+- **Repository:** https://github.com/duorc/duorc
+- **Paper:** https://arxiv.org/pdf/1804.07927.pdf
+- **Leaderboard:** https://duorc.github.io/#leaderboard
+- **Point of Contact:** [Needs More Information]
```
Comment (lhoestq) [inline datasets/duorc/README.md:57]:
```suggestion
- **Homepage:** [DuoRC](https://duorc.github.io/)
- **Repository:** [GitHub](https://github.com/duorc/duorc)
- **Paper:** [arXiv](https://arxiv.org/abs/1804.07927)
- **Leaderboard:** [DuoRC Leaderboard](https://duorc.github.io/#leaderboard)
- **Point of Contact:** [Needs More Information]
```

**Review by lhoestq — APPROVED** [review]
Thanks for adding it !

Looks all good to me :)

**Comment by gchhablani** [comment]
Thanks for approving @lhoestq!
Will apply these changes for the other datasets I've added too.

**Pull request merged**